pkgs/top-level/perl-packages.nix

```diff
@@ -8682,6 +8682,10 @@ let
       url = "mirror://cpan/authors/id/T/TO/TORBIAK/App-Git-Autofixup-0.003001.tar.gz";
       sha256 = "1q7im0zj238k5agwi7d1mz26a8r0wrxwfwp1l8n5k777gx3b5xhp";
     };
+    nativeBuildInputs = pkgs.lib.optional pkgs.stdenv.isDarwin pkgs.shortenPerlShebang;
+    postInstall = pkgs.lib.optionalString pkgs.stdenv.isDarwin ''
```

Comment:
```suggestion
    nativeBuildInputs = lib.optional stdenv.isDarwin pkgs.shortenPerlShebang;
    postInstall = lib.optionalString stdenv.isDarwin ''
```

Reply:
Oopsie, thanks!

Reply:
Would it be OK to `fixup!` this? (and force-push it)

Reply:
Yes pls. Also, it would be great if you could change the attribute name in the commit message from `perl-packages.GitAutoFixup` to `perlPackages.GitAutoFixup`

Reply:
Done and done!

> change perl-packages.GitAutoFixup to perlPackages.GitAutoFixup

I was a bit confused about what to name it, since we use it as (e.g.) `perl532Packages.***`, but this would also change the the other perl versions as well (e.g. `perl530Packages.***` )! 

Does the snake-case -> camelCase pattern apply to all top-level packages?

Reply:
`perlPackages` always points to the latest maint version of perl, like `perl532Packages`. I'm not sure there is a snake-case to camelCase rule tbh.